### ISSUE TITLE
Why using id ?

### ISSUE THREAD

**Issue by Shuunen** [LukyVj/Colorify.js#32]
Hi :)

All is in the title, why using id on line 29 : 
https://github.com/LukyVj/Colorify.js/blob/master/scripts/colorify.js#L29

And in the examples in doc :
![image](https://cloud.githubusercontent.com/assets/439158/20714997/0e2130c2-b64e-11e6-9507-13ab3a190eb1.png)
![image](https://cloud.githubusercontent.com/assets/439158/20715003/14fbecf2-b64e-11e6-8a45-e08e6a7fc1ee.png)
![image](https://cloud.githubusercontent.com/assets/439158/20715027/1fc42bae-b64e-11e6-802c-fbb9a573fc8b.png)

Regards.

**Comment by LukyVj** [comment]
Hey, thanks for the issue. 
Where ? 
For each gradient ? 


Just because it makes sense to me. But happy to discuss about it :)

**Comment by Shuunen** [comment]
yes i'm just wondering :) each call to colorify should have an id ?

**Comment by LukyVj** [comment]
Ok so, I will have to respond to that a bit later. I did this project nearly 1 year ago, and I barely put my hands in the code after that :p 

There were a V2 in it's way, but I stopped developing it. 
I have to first dig in my code, and then I'll respond. 

But so far, what I can say is I target each 'gradient's target' with ID's.

**Comment by Shuunen** [comment]
ok :p 

it's not a big deal, I was just curious,

Thanks for your lib anyway :+1:
